app/embedder.py

```diff
@@ -136,6 +156,9 @@ def embed():
     data = request.get_json()
 
     e = Embedder(data['bert'])
-    matrix = e.GetEmbedding(data['seq'])
-
-    return jsonify(str(len(matrix)))
+    if 'seq' in data:
```

Comment:
What are your thoughts on keeping the singluar `seq` and the plural `seq` keyword? We could just keep the plural one...

Reply:
Removed singular API